1559790879931	Sync.LogManager	DEBUG	Flushing file log
1559790879996	Sync.LogManager	DEBUG	Log cleanup threshold time: 1558926879996
1559790880109	Sync.LogManager	DEBUG	Done deleting files.
1559792758136	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559792758278	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559792758278	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559792758280	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559792758280	Sync.Service	INFO	Starting sync at 2019-06-06 05:45:58 in browser session DLDB2kz0MYXZ
1559792758280	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559792758282	Sync.Status	INFO	Resetting Status.
1559792758282	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1559792758762	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559792759042	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559792759042	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559792759043	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559792759699	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559792760018	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559792760018	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559792760024	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559792760024	Sync.Service	INFO	Starting sync at 2019-06-06 05:46:00 in browser session DLDB2kz0MYXZ
1559792760024	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559792760024	Sync.Service	INFO	Cannot start sync: already syncing?
1559792762774	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559794681154	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559794681292	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559794681293	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559794681293	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559794681293	Sync.Service	INFO	Starting sync at 2019-06-06 06:18:01 in browser session DLDB2kz0MYXZ
1559794681293	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559794681294	Sync.Service	INFO	Cannot start sync: already syncing?
1559794681760	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559794682192	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559794682192	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559794682193	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559794682733	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559794682938	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559794682938	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559794682940	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559794682940	Sync.Service	INFO	Starting sync at 2019-06-06 06:18:02 in browser session DLDB2kz0MYXZ
1559794682943	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559794682943	Sync.Service	INFO	Cannot start sync: already syncing?
1559794684110	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559794693660	Sync.Resource	WARN	GET request to https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections failed: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1559794693662	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1559794693668	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1559794693760	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
